docs/types.rst

```diff
@@ -352,6 +352,24 @@ check the value ranges at runtime and a failure causes an exception.  Enums need
         }
     }
 
+.. index:: ! contract type, ! type; contract
+
+.. _contract_types:
+
+Contracts
+---------
+
+Contracts themselves are a type that you can define and call from another
+contract, for example (where ``Token()`` is a contract) ::
```

Comment:
Sorry to be picky, this should be `Token` and not `Token()`.

Reply:
@axic OK, I think I was trying to be specific, but specified the wrong part :)